Massive HUD update

SolarLightTF2 opened the pull request:
* Added respawn counter with border. Anchored near middle of screen (copied that shit from solarlighthud-redux)
* Roundcounter is now smaller on minmode, doesn't conflict with respawn time
* Raised notifications on minmode
* Marked for death icon now uses the same skull icon that is seen in third person
* Advanced Options has been enlarged (no font changes, it depends on clientscheme and I'm waiting for resin's linux fixes)
* Added the ball countdown to minmode (I was listening to music very loud while editing the hud and missed my panacea)
* Added new voice chat speech bubbles based on the one used for typing

SolarLightTF2 committed:
New textures, borders, targetID, match hud, hudhealthaccount, many fixes

Not super thoroughly tested, I want to go to sleep

New borders and textures added. Affects some places indirectly, like the winpanel
New TargetID
New hudhealthaccount (+health when picking up HP or regenning HP)
New notifications (press F to inspect). Couldn't quite get the team textures to load without applying the rounded corner/outline shit so I just used a flat color texture for the team colors

Updated timer to not break with tf_use_match_hud 0
Updated waiting for players text, now matches setup
Updated pass/lockon notifications to be smaller and match the score display
Moved and reduced size of killfeed to prevent conflicts and added a background to improve readability

Animation overrides added. Edit scripts/hudanimations_pf.txt to override animations from hudanimations_tf.txt.
Disabled screenshot reminder popup animation during deathcam
Disabled match hud respawn animation (clearly not supposed to be there, plus it's buggy even in vanilla TF2. Most huds kill it)

Match hud updates:
Player panels tall value increased to make room for respawn count which now displays properly
No longer breaks in 12v12, but 6v7 looks odd, not much I can do without code editing
Moved round counter next to the timer

Fixed cutoff on scoreboard "server time left" text

kindredtf committed:
Merge branch 'development' into solar-hud-update

SolarLightTF2 committed:
bigger advanced options + more hud improvements

Added respawn counter with border, anchored near middle of screen (copied that shit from solarlighthud)
Roundcounter is now smaller on minmode, doesn't conflict with respawn time
Raised notifications on minmode
Marked for death icon now uses the same skull icon that is seen in third person
Advanced Option has been enlarged (no font changes, it depends on clientscheme and I'm waiting for resin's linux fixes)
Added the ball countdown to minmode (I was listening to music very loud while editing the hud and missed my panacea)
Added new voice chat speech bubbles based on the one used for typing

SolarLightTF2 committed:
Merge branch 'development' into solar-hud-update

Signed-off-by: SolarLight <40128400+SolarLightTF2@users.noreply.github.com>

SolarLightTF2 commented:
idk why it's trying to merge 3 commits but the only conflicts were 3 hud res files so they were easy to override

SolarLightTF2 commented:
<img width="847" height="597" alt="Screenshot 2025-08-15 192556" src="https://github.com/user-attachments/assets/a2274f61-368b-48b9-8ff9-9bf441dd509a" />
wtf. how. is this laced? some else test just in case

can't seem to reproduce

SolarLightTF2 committed:
Merge branch 'development' into solar-hud-update

CarlJosell committed:
Merge branch 'development' into solar-hud-update

CarlJosell reviewed (approved):
All looked good and worked fine. Couldn't produce the voicerecord bubbles, i guess this would be because im in a singleplayer server?... But they should work as normal since it's just icon changes.

SolarLightTF2 committed:
fix HP cutoff when using very high HP values

SolarLightTF2 committed:
fixed low ammo flashing animation when minmode is disabled

SolarLightTF2 committed:
fixed RED score count BG not being opaque in minmode

also added some vmts to solid color textures for purposes like this

SolarLightTF2 committed:
fix console errors

SolarLightTF2 committed:
marked for death icon no longer becomes toaster at low texture settings

SolarLightTF2 committed:
replaced voice bubble textures. updated voice chat and removed mipmaps from match hud dead class icons

SolarLightTF2 committed:
Updated scoreboard bottom stats to clarify that "assists" mean "kill-assists" and not goal assists

SolarLightTF2 committed:
new medic call bubbles

SolarLightTF2 committed:
updated freezepanel + hid screenshot reminder

SolarLightTF2 committed:
Scoreboard: PFGreen highlight on local player's name

SolarLightTF2 committed:
hopefully hidden statpanel (on the bright side)

SolarLightTF2 committed:
make two tips more beginner-friendly (what is a pogo)

SolarLightTF2 committed:
Adjusted keybind options and added missing text

SolarLightTF2 committed:
Merge branch 'development' into solar-hud-update

SolarLightTF2 commented:
bruh I didn't merge this in september? what the helli

**Old changes that never got merged because I forgor:**
Added spectator respawn counter with border. Anchored near middle of screen (copied that shit from solarlighthud-redux)
Roundcounter is now smaller on minmode, doesn't conflict with respawn time
Raised notifications on minmode
Marked for death icon now uses the same skull icon that is seen in third person
Advanced Options has been enlarged (no font changes, it depends on clientscheme and I'm waiting for resin's linux fixes)
Added the ball countdown to minmode (I was listening to music very loud while editing the hud and missed my panacea)
Added new voice chat speech bubbles based on the one used for typing

**New stuff**
Fixes hp and ammo jank
Fixes console errors
Adds a few new vmts/vtfs
New voice+medic bubbles +bugfix
Scoreboard update
Freezepanel update
Screenshot reminder removal
Fixes low res hud textures when using low texture settings
Hide "on the bright side" messages
Adjusted keybind options slightly
Minor chat_english edits
Probably something else I'm forgetting

SolarLightTF2 committed:
fix scoreboard regression

SolarLightTF2 committed:
change game name to "PASS Fortress"

SolarLightTF2 committed:
New Crit and Mini-Crit pop-up text

SolarLightTF2 committed:
moved killstreak slightly. changed zpos of class dual-line bg to go behind class

SolarLightTF2 committed:
lowered spectator respawn to account for tournament mode team names

SolarLightTF2 committed:
New damage indicators

SolarLightTF2 committed:
Sourcescheme update

SolarLightTF2 committed:
Sourcescheme - change keybind menu highlight from orange to green

SolarLightTF2 committed:
updated keyboard menu to say pass fortress for real this time

SolarLightTF2 committed:
sourcescheme - crosshair selector bg

SolarLightTF2 committed:
update targetid + targetid HP

SolarLightTF2 committed:
new healthiconpanel

SolarLightTF2 committed:
new loading spinner

SolarLightTF2 committed:
Merge branch 'development' into solar-hud-update

SolarLightTF2 committed:
perfectly balanced hudplayerhealth hp value

SolarLightTF2 committed:
Merge branch 'development' into solar-hud-update

SolarLightTF2 committed:
first draft of team select menu

SolarLightTF2 committed:
Team select second draft

SolarLightTF2 committed:
remove unused font definition (team select)

SolarLightTF2 committed:
add spectator TV sound effects, replace round start siren, add game_sounds.txt

SolarLightTF2 committed:
siren no longer destroys ears

SolarLightTF2 committed:
add medic ubercharge (flaresh)

SolarLightTF2 committed:
update teammenu with new random sign

SolarLightTF2 committed:
new damage indicator (again)

SolarLightTF2 committed:
medic ubercharge update for consistency

SolarLightTF2 committed:
Add Tournament Mode (flaresh)

SolarLightTF2 committed:
Hid equipped item labels and hid sync loadout button

SolarLightTF2 committed:
Added sepia shader, modulate texture, updated mapinfo and motd screens

SolarLightTF2 committed:
replace useless motd text with lore dump

Signed-off-by: SolarLight <40128400+SolarLightTF2@users.noreply.github.com>

SolarLightTF2 committed:
Update mapinfo gamemode guide to match new ball controls

SolarLightTF2 committed:
Merge branch 'solar-hud-update' of https://github.com/p4sstime/p4ss into solar-hud-update

SolarLightTF2 committed:
Update motd_default.txt

SolarLightTF2 committed:
Update chat_english.txt

SolarLightTF2 committed:
sepia shader no longer does weirdness nearby walls, but now hides viewmodel (thanks cueki)

SolarLightTF2 committed:
Merge branch 'development' into solar-hud-update

prplnorangesoda reviewed (approved):
LGTM
